.github/workflows/npm-package-updater.yaml

```diff
@@ -0,0 +1,68 @@
+name: Update NPM Package
+
+on:
+  workflow_dispatch:
+  repository_dispatch:
+    types:
+      - update-package
+
+jobs:
+  update-package:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 14
+
+      - name: Git configuration
+        run: |
+          git config --global user.email "bot@sunodo"
+          git config --global user.name "GitHub Actions release workflow"
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Get latest package information
+        id: get-package
+        run: |
+          PACKAGE_NAME="@sunodo/cli"
+          PACKAGE_VERSION=$(npm show $PACKAGE_NAME version)
+          PACKAGE_URL=$(npm show $PACKAGE_NAME dist.tarball)
+          npm pack @sunodo/cli@$PACKAGE_VERSION
+          PACKAGE_SHA256=$(sha256sum sunodo-cli-$PACKAGE_VERSION.tgz | awk '{print $1}')
+
+          if [ -z "$PACKAGE_VERSION" ] || [ -z "$PACKAGE_URL" ] || [ -z "$PACKAGE_SHA256" ]; then
+            echo "Failed to retrieve package information. Exiting..."
+            exit 1
+          fi
+
+          echo "PACKAGE_VERSION=$PACKAGE_VERSION" >> $GITHUB_ENV
+          echo "PACKAGE_URL=$PACKAGE_URL" >> $GITHUB_ENV
+          echo "PACKAGE_SHA256=$PACKAGE_SHA256" >> $GITHUB_ENV
```

Comment:
Although the hash is the way to check for integrity, defining the hash for the package should come from the package that's already published since this step could have problems during download and we could add the wrong hash.

So, we should use the `npm view @sunodo/cli dist.integrity` that's a bse64 encoded SHA512.

So the `Formula/sunodo.rb` should use `sha512` instead.

Reply:
Note: I'm not sure if homebrew Formula supports `sha512`. Have tom check.

Reply:
In case homebrew only supports sha256 we should check the download against sha512 before running sha256 to change the sunodo.rb Formula.

Reply:
It looks, it does not support.
https://github.com/Homebrew/legacy-homebrew/issues/38945